presto-server-rpm/pom.xml

```diff
@@ -0,0 +1,177 @@
+            <version>${project.version}</version>
+            <type>tar.gz</type>
+            <scope>runtime</scope>
+        </dependency>
+    </dependencies>
+
+    <profiles>
+        <profile>
+            <id>rpmbuild</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <!-- Untar presto-server tgz to target build folder -->
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <version>2.5</version>
+                        <executions>
+                            <execution>
+                                <id>unpack</id>
+                                <goals>
+                                    <goal>unpack</goal>
+                                </goals>
+                                <configuration>
+                                    <artifactItems>
+                                        <artifactItem>
+                                            <groupId>com.facebook.presto</groupId>
+                                            <artifactId>presto-server</artifactId>
+                                            <version>${project.version}</version>
+                                            <type>tar.gz</type>
+                                            <outputDirectory>${project.build.directory}</outputDirectory>
+                                        </artifactItem>
+                                    </artifactItems>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <!-- Build presto-server rpm using the untarred artifacts -->
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>rpm-maven-plugin</artifactId>
+                        <version>2.1.2</version>
+                        <executions>
+                            <execution>
+                                <id>generate-rpm</id>
+                                <goals>
+                                    <goal>rpm</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <name>presto</name>
+                            <version>${project.version}</version>
+                            <release>1.0</release>
+                            <group>Applications</group>
+                            <description>Presto Server RPM Package.</description>
+                            <needarch>x86_64</needarch>
+                            <requires>
+                                <require>python >= 2.6</require>
```

Comment:
This was originally added for presto-admin. But it makes sense to keep that requirement separate from that of rpm. So updated it to have 2.4 instead.